libs/island-ui/core/src/lib/Button/Button.stories.tsx

```diff
@@ -1,132 +0,0 @@
-import React from 'react'
```

Comment:
I would keep the story with the deprecation message until no project relies on the component anymore

Reply:
It's confusing to have Button and ButtonRC stories next to each other. Just have a deprecated message on the component itself and show new developers/users the correct story straight away.